steps/report.sh

```diff
@@ -48,7 +48,40 @@ if [ ! -s "${list}" ]; then
     exit 1
 fi
 
-sort -o "${list}" "${list}"
+groups=($(grep -oP '\[.*?\]' "${list}" | sed 's/[][]//g') "")
+
+st_list=${TARGET}/temp/structured-list-of-metrics.tex
+rm -f "${st_list}"
+touch "${st_list}"
+
+for group in "${groups[@]}"; do
+
```

Comment:
@Raleksan what's the point of placing empty lines so often? Read this please: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

Reply:
@yegor256 

I read the article and I think your point makes sense.

My style is from hardware development (our code usually looks like [that](https://github.com/MPSU/SoCDC_RTL_2024/blob/main/rtl/liteic_slave_node_read.sv)), and structuring through spaces is our code style. 

I will pay more attention to this point.